### PR TITLE
Fixed ib_pingpong test case

### PR DESCRIPTION
**Pull request by smruti77**
Fixted ib_pingpong.py.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
JOB ID     : ee2aac6c80df59e914de98c563b2df6d578d6877
JOB LOG    : /root/avocado/job-results/job-2019-09-24T03.18-ee2aac6/job.log
 (1/4) ib_pingpong.py:PingPong.test_ib_pingpong;ud_pingpong-parameters-e248: PASS (19.23 s)
 (2/4) ib_pingpong.py:PingPong.test_ib_pingpong;uc_pingpong-parameters-7279: PASS (65.94 s)
 (3/4) ib_pingpong.py:PingPong.test_ib_pingpong;rc_pingpong-parameters-3e82: PASS (66.28 s)
 (4/4) ib_pingpong.py:PingPong.test_ib_pingpong;srq_pingpong-parameters-c33f: PASS (50.02 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 201.94 s
JOB HTML   : /root/avocado/job-results/job-2019-09-24T03.18-ee2aac6/results.html

File: io/net/infiniband/ib_pingpong.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def setUp(self):
         self.peer_iface = ''
         cmd = "ip addr show"
         output, exitcode = self.run_command(cmd)
-        for line in output.splitlines():
+        for line in str(output).splitlines():
```
Comment (narasimhan-v) [inline io/net/infiniband/ib_pingpong.py]: decode('utf-8') instead of str() please

Comment (smruti77) [reply io/net/infiniband/ib_pingpong.py]: i tried with decode('utf-8') but i am geeting error," ERROR: 'list' object has no attribute 'decode' " so i kept like this.

Comment (narasimhan-v) [reply io/net/infiniband/ib_pingpong.py]: you need to decode each element in the list

**Comment by smruti77** [comment]
handling this issue with passwordless ssh.

**Pull request closed without merging**